.github/workflows/test-upgrade.yml

```diff
@@ -105,12 +105,10 @@ jobs:
 
       - name: Run the upgrade tests
         # NOTE: "PATH=$PATH" preserves the default user $PATH. This is needed to maintain the version of zarf installed
-        #       in a previous step. This test run will the current release to create a K3s cluster.
+        #       in a previous step. This test run will use the current release to create a K3s cluster.
         run: |
           sudo env "PATH=$PATH" CI=true zarf tools kubectl describe nodes
 
-          zarf package create src/test/upgrade --set PODINFO_VERSION=6.3.4 --confirm
```

Comment:
Not sure why this is removed - we should be able to deploy old packages with new Zarf versions (which this tests)